pkg/types/constants.go

```diff
@@ -25,4 +25,8 @@ const (
 	ApproleAuth              = "approle"
 	GithubAuth               = "github"
 	IAMAuth                  = "iam"
+
+	AVPPathAnnotation   = "avp.kubernetes.io/path"
+	AVPIgnoreAnnotation = "avp.kubernetes.io/ignore"
+	KVVersionAnnotation = "avp.kubernetes.io/kv-version"
```

Comment:
Doesn't really matter but `VaultKVVersionAnnotation` might be a better name especially since that only applies to Vault and this plugin now supports non-Vault things